pkg/rtc/config.go

```diff
@@ -118,20 +119,55 @@ func NewWebRTCConfig(conf *config.Config) (*WebRTCConfig, error) {
 			PacketBufferSizeAudio: rtcConf.PacketBufferSizeAudio,
 		},
 		Publisher:  publisherConfig,
-		Subscriber: getSubscriberConfig(rtcConf.CongestionControl.UseSendSideBWEInterceptor || rtcConf.CongestionControl.UseSendSideBWE),
+		Subscriber: getSubscriberConfig(types.CurrentProtocol, rtcConf.CongestionControl.UseSendSideBWEInterceptor || rtcConf.CongestionControl.UseSendSideBWE),
 	}, nil
 }
 
-func (c *WebRTCConfig) UpdateCongestionControl(ccConf config.CongestionControlConfig) {
-	c.Subscriber = getSubscriberConfig(ccConf.UseSendSideBWEInterceptor || ccConf.UseSendSideBWE)
+func (c *WebRTCConfig) UpdateSubscriberConfig(protocolVersion types.ProtocolVersion, ccConf config.CongestionControlConfig) {
+	c.Subscriber = getSubscriberConfig(protocolVersion, ccConf.UseSendSideBWEInterceptor || ccConf.UseSendSideBWE)
 }
 
 func (c *WebRTCConfig) SetBufferFactory(factory *buffer.Factory) {
 	c.BufferFactory = factory
 	c.SettingEngine.BufferFactory = factory.GetOrNew
 }
 
-func getSubscriberConfig(enableTWCC bool) DirectionConfig {
+func getSubscriberConfig(protocolVersion types.ProtocolVersion, enableTWCC bool) DirectionConfig {
+	if protocolVersion.SupportsSinglePeerConnection() {
```

Comment:
As `subscriber` peer connection is used for both directions, these become union of publisher and subscriber config and set on the media engine.